### DISCUSSION TITLE
Add ability for all buttons to return json results...not just button 0

### DISCUSSION THREAD

**Discussion by LtCmdrScott** [swiftDialog/swiftDialog#359] [Ideas]
An added feature would be nice to have any button press return selection results (not just button 0).  for some reason, I cannot upload my image for reference, but I have a backup / restore routine that has someone choose a file location (Network, USB, etc) (radio button selection) and then button 0 is "backup" and button 1 is "restore", but obviously if they choose button 1, no results are returned as it is being treated as an Esc / Cancel

**Comment by wakco** [comment]
I'd add Radio buttons or a menu for Backup/Restore selection, then have button 0 be either Go, or Okay, and button 1 can then still be Cancel.

In the end I'd have to ask, why would we be forcing the user to choose only restore or backup, i.e. why is neither not an option?
